CODEOWNERS

```diff
@@ -201,6 +201,7 @@
 /boards/arm64/fvp_baser_aemv8r/           @povergoing
 /boards/arm64/fvp_base_revc_2xaemv8a/     @carlocaione
 /boards/arm64/intel_socfpga_agilex_socdk/ @siclim @ngboonkhai
+/boards/arm64/intel_socfpga_agilex5_socdk/ @chongteikheng
```

Comment:
ditto

Reply:
Sure, will update here also.

Reply:
Created a PR for the same. https://github.com/zephyrproject-rtos/zephyr/pull/65190